src/reporting/disputeBondToken.se

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+
+#### must be first data members to match delegator.se
+data controller
+data controllerLookupName
+####
+
+inset('../macros/externs.sem')
+inset('../macros/safeMath.sem')
+inset('../macros/assertNoValue.sem')
+inset('../macros/require.sem')
+
+data initialized
+data market
+data bondHolder
+data payoutDistributionHash #Disputed outcome
+data bondRemainingToBePaidOut
+data tokens[2**160]
+data spenders[2**160][2**160]
+data supply
+
+def any():
+    assertNoValue()
+
+def initialize(market: address, bondHolder: address, bondAmount: int256, payoutDistributionHash: int256):
+    require(not self.initialized)
+    self.initialized = 1
+    self.market = market
+    self.bondHolder = bondHolder
+    self.payoutDistributionHash = payoutDistributionHash
+    self.bondRemainingToBePaidOut = bondAmount * 2
+    self.market.getReputationToken().transferFrom(msg.sender, self, bondAmount)
+
+def withdraw(amountToWithdraw: int256, payoutDistributionHash: int256):
+    require(msg.sender == self.bondHolder)
+    require(self.market.isFinalized())
+    require(payoutDistributionHash == self.payoutDistributionHash)
+    if (self.bondRemainingToBePaidOut > 0):
```

Comment:
Ok, so I should just get rid of the amountToWithdraw parameter and change amountToTransfer:

amountToTransfer = self.market.getReputationToken().balanceOf(self)

Correct?